backend/services/implementations/__tests__/testService.test.ts

```diff
@@ -133,4 +133,33 @@ describe("mongo testService", (): void => {
     assertResponseMatchesExpected(test, originalTest);
     expect(test.id).toEqual(originalTest.id);
   });
+
+  it("unarchiveTest - success", async () => {
+    const test = await MgTest.create({
+      ...mockTest,
+      status: AssessmentStatus.ARCHIVED,
+    });
+
+    const unarchivedTest = await testService.unarchiveTest(test.id);
+    assertResponseMatchesExpected(
+      {
+        ...mockTest,
+        status: AssessmentStatus.DRAFT,
+      },
+      unarchivedTest,
+    );
+    expect(test.id).not.toEqual(unarchivedTest.id);
+
+    const originalTest = await MgTest.findById(test.id);
+    // TODO: update this to be soft delete instead of hard delete
```

Comment:
follow up ticket for once brian merges his PR https://www.notion.so/uwblueprintexecs/Update-unarchiveTest-unit-test-40608ab551c64370a6837e3d7d122961?pvs=4